Gir.toml

```diff
@@ -43,7 +41,6 @@ generate = [
     "Gio.SettingsBindFlags",
     "Gio.SettingsSchema",
     "Gio.SimpleActionGroup",
-    "Gio.SimpleIOStream",
```

Comment:
Why do you drop this one? It's not very important for bindings in any case

Reply:
It just moved as object to hide properties